addons/acr/CfgVehicles.hpp

```diff
@@ -0,0 +1,157 @@
+        magazines[] = {};
+        respawnMagazines[] = {};
+        items[] = {};
+        respawnItems[] = {};
+        linkedItems[] = {};
+        respawnLinkedItems[] = {};
+        accuracy = 2.3;
+        camouflage = 1.2;
+        cost = 90000;
+        threat[] = {1,0.1,0.1};
+        class Wounds {
+            tex[] = {};
+            mat[] = {
+                "A3\Characters_F\BLUFOR\Data\clothing1.rvmat",
+                "A3\Characters_F\BLUFOR\Data\clothing1_injury.rvmat",
+                "A3\Characters_F\BLUFOR\Data\clothing1_injury.rvmat",
+                "A3\Characters_F\Common\Data\basicbody.rvmat",
+                "A3\Characters_F\Common\Data\basicbody_injury.rvmat",
+                "A3\Characters_F\Common\Data\basicbody_injury.rvmat",
+                "A3\Characters_F\Common\Data\coveralls.rvmat",
+                "A3\Characters_F\Common\Data\coveralls_injury.rvmat",
+                "A3\Characters_F\Common\Data\coveralls_injury.rvmat",
+                "A3\Characters_F\Heads\Data\hl_white_bald_muscular.rvmat",
+                "A3\Characters_F\Heads\Data\hl_white_bald_muscular_injury.rvmat",
+                "A3\Characters_F\Heads\Data\hl_white_bald_muscular_injury.rvmat",
+                "A3\Characters_F\Heads\Data\hl_black_bald_muscular.rvmat",
+                "A3\Characters_F\Heads\Data\hl_black_bald_muscular_injury.rvmat",
+                "A3\Characters_F\Heads\Data\hl_black_bald_muscular_injury.rvmat",
+                "A3\Characters_F\Heads\Data\hl_white_hairy_muscular.rvmat",
+                "A3\Characters_F\Heads\Data\hl_white_hairy_muscular_injury.rvmat",
+                "A3\Characters_F\Heads\Data\hl_white_hairy_muscular_injury.rvmat",
+                "A3\Characters_F\Heads\Data\hl_white_old.rvmat",
+                "A3\Characters_F\Heads\Data\hl_white_old_injury.rvmat",
+                "A3\Characters_F\Heads\Data\hl_white_old_injury.rvmat",
+                "A3\Characters_F\Heads\Data\hl_asian_bald_muscular.rvmat",
+                "A3\Characters_F\Heads\Data\hl_asian_bald_muscular_injury.rvmat",
+                "A3\Characters_F\Heads\Data\hl_asian_bald_muscular_injury.rvmat"
+            };
+        };
+    };
+
+    #include "vehicles\CfgAntiAir.hpp"
+    #include "vehicles\CfgArtillery.hpp"
+    #include "vehicles\CfgBuran.hpp"
+    #include "vehicles\CfgBVP.hpp"
+    #include "vehicles\CfgDingo.hpp"
+    #include "vehicles\CfgDrone.hpp"
+    #include "vehicles\CfgHeli.hpp"
+    #include "vehicles\CfgJet.hpp"
+    #include "vehicles\CfgPandur.hpp"
+    #include "vehicles\CfgStatic.hpp"
+    #include "vehicles\CfgT72.hpp"
+    #include "vehicles\CfgTatra.hpp"
+    #include "units\CfgUnits.hpp"
+
+    class CUP_B_Kombat_Radio_Olive;
+    class GVAR(Squad_Leader_pack): CUP_B_Kombat_Radio_Olive {
+        scope = 1;
+        class TransportMagazines {
+            MACRO_ADDMAGAZINE(1Rnd_HE_Grenade_shell,11);
+        };
+        class TransportItems{};
+        class TransportWeapons{};
```

Comment:
If the base bag doesn't specify items or weapons, these lines are extraneous as the properties don't need overwriting